README.md

```diff
@@ -1,24 +1,75 @@
-[![tests](https://github.com/ddev/ddev-redis/actions/workflows/tests.yml/badge.svg)](https://github.com/ddev/ddev-redis/actions/workflows/tests.yml) ![project is maintained](https://img.shields.io/maintenance/yes/2024.svg)
+<div align="center">
 
-## What is this?
+# ddev-redis - Redis container for DDEV
 
-This repository allows you to quickly install Redis into a [DDEV](https://ddev.readthedocs.io) project using just `ddev get ddev/ddev-redis`.
+[![GitHub release (with filter)](https://img.shields.io/github/v/release/ddev/ddev-redis)](https://github.com/ddev/ddev-redis/releases)
+[![Tests](https://github.com/ddev/ddev-redis/actions/workflows/cron_tests.yml/badge.svg)](https://github.com/ddev/ddev-redis/actions/workflows/cron_tests.yml)
+![project is maintained](https://img.shields.io/maintenance/yes/2024.svg)
+
+</div>
+
+This repository provides [Redis](https://redis.com) container for [DDEV](https://ddev.readthedocs.io/).
+
+It is based on [redis:6-alpine](https://hub.docker.com/layers/library/redis/6-alpine) docker image and [DDEV custom compose files](https://ddev.readthedocs.io/en/stable/users/extend/custom-compose-files/)
+
+## Version 2 features
+
+There are a lot of differences between **v1** and **v2
+
+| Feature           | v1               | v2      |
+| ----------------- | ---------------- | ------- |
+| Maximum Memory    | Unlimited        | 512Mb   |
+| Persistence       | No               | **Yes** |
+| Image Size        | 113  Mb          | 30Mb    |
```

Comment:
30Mb is a stripped image from your add-on. `redis:7-alpline` takes 38Mb.
I think the important is what you download.
Let's compare the compressed size between `6-bullseye` and `6-alpine`.

```suggestion
| Image Size        | ~40Mb            | ~11Mb   |
```